src/urllib3/util/ssl_.py

```diff
@@ -237,9 +237,7 @@ def resolve_ssl_version(candidate):
     return candidate
 
 
-def create_urllib3_context(
-    ssl_version=None, cert_reqs=None, options=None, ciphers=None
-):
+def create_hip_context(ssl_version=None, cert_reqs=None, options=None, ciphers=None):
```

Comment:
What are your thoughts on using `create_ssl_context()` instead of having the package name in the function name?